### PR TITLE
catch file not found errors when checking destination resource

### PR DESCRIPTION
**Pull request by jwestgard**
When iterating over a tree of original resources and checking against the destination path, previously the tool would raise an uncaught exception when a file had been deleted or was otherwise missing from the destination tree. This change wraps that process in a "try/except" construct in order to handle missing files gracefully (recording the error, but not breaking off verification).

Nota bene: Because of indentation changes, the diff below looks like a lot more is changing than is in fact the case.

### CODE REVIEW

File: verify.py [hunk 1]
```diff
@@ -430,34 +430,40 @@ def credentials(user):
...
+                        verification = original.sha1
+                    else:
+                        verified = False
+                        verification = "{0} != {1}".format(
+                            original.sha1, destination.sha1
+                            )
+                elif original.type == "rdf":
+                    if isomorphic(original.graph, destination.graph):
+                        verified = True
+                        verification = "{0} triples".format(len(original.graph))
```
Comment (dbernstein) [inline verify.py]: looks like there was a line length warning from travis : " line too long (80 > 79 characters)"

**Comment by awoods** [comment]
@jwestgard : according to @dbernstein, this PR is ready to go once the travis error has been resolved.

**Comment by jwestgard** [comment]
Thanks @awoods . I will fix and rebase shortly.

**Comment by jwestgard** [comment]
Ready for merge. Turns out there were no conflicts -- perhaps those will come in other outstanding PRs?

**Comment by awoods** [comment]
You should have commit privileges now, @dbernstein .

**Comment by awoods** [comment]
@dbernstein : in cases like this, where there are multiple commits due to code review iteration and those commits should really be squashed into one... please commit this by using the "Squash and merge" button.

**Pull request merged**